src/Field/CoordinateReferenceSystemCombo/CoordinateReferenceSystemCombo.less

```diff
@@ -0,0 +1,3 @@
+.react-geo-crs-combo {
+  width: 200px;
```

Comment:
why do we have a fixed width in the library?

Reply:
removed less file in 77710baa1d8. Style can be passed as prop now.